### ISSUE TITLE
Fail to save Vod to NTFS HDD

### ISSUE THREAD

**Issue by Frambooisier** [mohad12211/twitch-downloader-gui#6]
saving files to BTRFS main SSD and USB VFAT works fine but when I try to save VOD to HDD NTFS I got following error:
```
[NULL @ 0x55b25b9ae5c0] Unable to find a suitable output format for '/mnt/WDJ/Recordings/DF/I'
/mnt/WDJ/Recordings/DF/I: Invalid argument

```

**Comment by mohad12211** [comment]
Can you please try to execute the same thing but with the `TwitchDownloaderCLI`?

**Comment by Frambooisier** [comment]
yes, indeed its a TwitchDownloaderCLI problem

```
[STATUS] - Downloading 100% (1/3)
[STATUS] - Combining Parts (2/3)
[STATUS] - Finalizing MP4 (3/3)
[NULL @ 0x557f7af8a2c0] Unable to find a suitable output format for '/mnt/WDJ/Recordings/'
/mnt/WDJ/Recordings/: Invalid argument


```

**Comment by Frambooisier** [comment]
I have opened the issue on the TwitchDownloader github.

BTW thank you for the beatiful GUI! @mohad12211

**Comment by Frambooisier** [comment]
@mohad12211 yes, the error was displayed in the log box inside the app, and the app did not crash

**Comment by Frambooisier** [comment]
If I would have some feature request I would love to be able to post them somewhere here :)

**Comment by Frambooisier** [comment]
Im not sure, what exacly happened, but one option could be that I named the output file with spaces like "this is my file" instead of "thisismyfile", so I assume the name gets passed to the TwitchDownloader command without "xxx" or my \ file.

**Comment by Frambooisier** [comment]
well same error happens when picking path that have directory with white spaces.
[NULL @ 0x56435a5ed600] Unable to find a suitable output format for '/mnt/WDJ/Recordings/DF/DF'
/mnt/WDJ/Recordings/DF/DF: Invalid argument

**Comment by mohad12211** [comment]
yes, I forgot to add quotation marks for the output files, fixed it here b7ff277bffd6421e8caad9f3cd17ecbbd74605ad, will be alright with the next release.
